### PR TITLE
Derive PartialEq+Eq.

### PR DESCRIPTION
**Pull request by rickvanprim**
Fixes https://github.com/rust-random/rand/issues/964.

### CODE REVIEW

**Comment by rickvanprim** [comment]
Note this is dependent on https://github.com/rust-random/rand/pull/975.

File: rand_jitter/src/lib.rs [hunk 1]
```diff
@@ -122,6 +122,7 @@ const MEMORY_SIZE: usize = MEMORY_BLOCKS * MEMORY_BLOCKSIZE;
 ///
 /// Note that this RNG is not suitable for use cases where cryptographic
 /// security is required.
+#[derive(PartialEq, Eq)]
```
Comment (dhardy) [inline rand_jitter/src/lib.rs:125]: Not this one — it's non-deterministic.

Comment (rickvanprim) [reply rand_jitter/src/lib.rs:125]: Good catch, will fix.

**Comment by dhardy** [comment]
Uh, there are two more changes to the `rand_jitter` crate that you didn't revoke there. You might as well squash those changes.

**Comment by rickvanprim** [comment]
Sorry about that, I'll revert them.  Is maintaining a single squashed commit and force pushing an acceptable workflow?  It would certainly make it easier for me to keep these things from slipping through, but I don't know if that's discouraged.

**Comment by dhardy** [comment]
Yes, squashing is fine with me (provided you're not making massive commits).

**Comment by vks** [comment]
Implemented by #7.

**Pull request closed without merging**